misc/navigation.py

```diff
@@ -3,19 +3,18 @@
 ctx = Context("navigation")
 
 keymap = {
+    # Requires activation of System Preferences -> Shortcuts -> Input Sources
+    # -> "Select the previous input source"
+    "change language": Key("ctrl-space"),
+    # moving
+    "tarsh": Key("shift-tab"),
     # scrolling
     "scroll down": [Key("down")] * 30,
     "scroll up": [Key("up")] * 30,
     # NB home and end do not work in all apps
     "(scroll way down | doomway)": Key("cmd-down"),
     "(scroll way up | jeepway)": Key("cmd-up"),
-    "page up": [Key("pageup")],
```

Comment:
page up and page down can be removed from here :+1:

Reply:
Removed these commands in favor of the commands in basic keys.